docs/docx_templates.md

```diff
@@ -48,10 +83,400 @@ For example:
+:::
+
+### Use `{%p %}` and `{%tr %}` to control white space at the paragraph level
+
+When inserting conditional paragraphs, make sure that they are wrapped in `{%p %}` tags, like this:
+
+```jinja
+{%p if some_condition %}
+This is my conditional paragraph.
+{%p endif %}
+```
+
+But make sure that the `{%p %}` is truly in its own paragraph,
+not just a soft carriage return. You can check by toggling the [show formatting marks](https://support.microsoft.com/en-us/office/turn-formatting-marks-on-or-off-b166e811-d762-4f24-8328-d897cdace459) option in Word.
+
+Also use `{%p if %}` to control conditional items in a list, like this:
+
+1. `{%p if condition %}`
+1. Conditional text
+1. `{%p endif %}`
+
+Keep the condition inside the list itself. Avoid using the condition in its own 
+paragraph, as when Docassemble merges the two lists they may behave unpredictably.
+
+Similarly, use `tc` and `tr` to control repeated or conditional rows.
+Keep this in-line in one table when it makes sense. Avoid using `{%p %}` and multiple tables, as this can lead to unexpected table formatting when the two
+separate tables are merged.
+
+For example, to repeat rows in a table:
+
+Attribute 1 | Attribute 2
+--------------------|-------------------|
+ `{%tr for item in my_list %}` | &nbsp;|
+ `{{ item.attribute_1 }}` | `{{ item.attribute_2 }}` |
+ `{%tr endfor %}` | &nbsp;
+
+The same `{%tr %}` can be used for conditional rows, and `{%tc %}` works
+similarly for conditional or repeated columns.
+
+`{%tr %}` and `{%tc %}` should always be in their own row or column.
+
+:::caution Don't use `{%p %}` inside a table cell
+`{%p %}` can lead to unexpected results inside a table. Use `{%tr %}` instead.
+:::
+
+### Use `{% if %}` for conditional text inside a single paragraph
+
+If you have a few conditional words in a paragraph, you can use `{% if %}`
+in-line in the paragraph. For example:
+
+`{%if user == "self" %}I request{% else %}{{user}} requests{% endif %} the following:`
+
+If you have many repeated examples of the same condition, though, you may be
+better off making the whole paragraph conditional.
+
+When using in-line conditions, make sure you consistently place the space across
+each condition.
+
+### Use `output_checkbox()` for conditional checkbox fields that look like paper forms
+
+In some cases, you need to make your Word Document look like a document that was
+filled in by hand. [`output_checkbox()`](https://suffolklitlab.org/docassemble-AssemblyLine-documentation/docs/framework/altoolbox#shorthand-function-to-display-a-checkbox-in-replace-of-a-truefalse-boolean-value-in-a-docx-template) can be used to add a checkbox in-line in your document.
+
+These two expressions are equivalent, but the second version takes substantially less
+space in your template:
+
+Using if/else/endif | Using output_checkbox()
+--------------------|------------------------------
+`{%if condition %}[X]{% else %}[  ]{% endif %}` | `{{ output_checkbox(condition) }}`
```

Comment:
I think it would be useful to mention that this is an AL feature, not a da or Jinja one